### PR TITLE
Fix animations when you are on the last page

### PR DESCRIPTION
**Pull request by vadymmarkov**
Should fix https://github.com/hyperoslo/Presentation/issues/40

### CODE REVIEW

File: Source/PresentationController.swift [hunk 1]
```diff
@@ -142,8 +142,9 @@ extension PresentationController: UIScrollViewDelegate {
     let offset = scrollView.contentOffset.x - CGRectGetWidth(view.frame)
     let offsetRatio = offset / CGRectGetWidth(view.frame)
 
+
     var index = animationIndex
-    if (offsetRatio > 0.0 && index < pagesCount - 1) || (index == 0) {
+    if (offsetRatio > 0.0) || (index == 0) {
```
Comment (3lvis) [inline Source/PresentationController.swift]: Do you need the parenthesis?

Would this work?

```swift
if offsetRatio > 0.0 || index == 0 {
```

File: Source/PresentationController.swift [hunk 2]
```diff
@@ -152,6 +153,8 @@ extension PresentationController: UIScrollViewDelegate {
       !(index == pagesCount)
 
     if canMove {
+      println("\(index) - \(pagesCount)")
```
Comment (3lvis) [inline Source/PresentationController.swift]: Can you remove this?

File: Source/PresentationController.swift (hunk 1 above)
Comment (vadymmarkov) [inline Source/PresentationController.swift]: Right, now we can.

File: Source/PresentationController.swift (hunk 2 above)
Comment (vadymmarkov) [inline Source/PresentationController.swift]: Yes, thanks.

**Comment by vadymmarkov** [comment]
@3lvis Fixed things.

**Pull request merged**